core/kazoo_amqp/src/gen_listener.erl

```diff
@@ -554,9 +566,11 @@ handle_cast({'resume_consumers'}, #state{is_consuming='false'
                                         ,params=Params
                                         ,queue=Q
                                         ,other_queues=OtherQueues
+                                        ,auto_ack=AutoAck
                                         }=State) ->
-    start_consumer(Q, props:get_value('consume_options', Params)),
-    _ = [start_consumer(Q1, props:get_value('consume_options', P))
+    ConsumeOptions = props:get_value('consume_options', Params, []),
```

Comment:
I wonder if we can just prepend the main queue into `OtherQueues` and run the LC versus having it separated out like this? Something like:

```erlang
[start_consumer(...) || {Queue, {_, QueueParams}} <- [{Q, {whatever, Params}} | OtherQueues]]
```

Thoughts?